tests/sdl/test_model.py

```diff
@@ -0,0 +1,96 @@
+import opengeode
+from opengeode import ogAST
+from sdl2promela.sdl.model import Model, NextState, Output
+import os
+
+TEST_DIR: str = os.path.dirname(os.path.realpath(__file__))
+RESOURCE_DIR: str = os.path.join(TEST_DIR, "resources")
+
+
+def read_main_process(path: str) -> ogAST.Process:
+    try:
+        ast, warnings, errors = opengeode.parse([path])
+        assert len(warnings) == 0
+        assert len(errors) == 0
+        assert len(ast.processes) == 1
+        return ast.processes[0]
+    except BaseException as error:
```

Comment:
baaaad idea

especially in test code - just let the test runner to catch the exception and print the stack trace

Reply:
My intent was not to test opengeode, but just get some info when I try to work with a broken input. But ok, let's remove the try except.